requirements.txt

```diff
@@ -1,10 +1,10 @@
 Django==2.2
 django-prometheus==1.0.15
-canonicalwebteam.blog==1.5.0
+canonicalwebteam.blog==1.5.1
 canonicalwebteam.custom_response_headers==0.2.0
 canonicalwebteam.django_views==1.3.4
 canonicalwebteam.get-feeds==0.2.4
-canonicalwebteam.http==0.1.6
+canonicalwebteam.http==1.0.1
```

Comment:
Try running the site with `./run --env SEARCH_API_KEY=fakekey` - you'll see that this version of `http` causes an error.

I think `webapp/views.py` line 25 just needs changing to use `fallback_cache_duration`.

Reply:
Great catch. The solution worked!